### ISSUE TITLE
API returns invalid GeoJSON

### ISSUE THREAD

**Issue by p1d1d1** [geoadmin/mf-geoadmin3#4526]
e.g. this request:

http://api3.geo.admin.ch/rest/services/api/MapServer/ch.swisstopo.swissboundaries3d-kanton-flaeche.fill/1?geometryFormat=geojson

is invalid according to http://geojsonlint.com/:

1) response structure is not correct
2) coordinates do not follow the right-hand rule (see GeoJSON specs)

As far as 1) is concerned, the API returns the following structure (I filtered the coordinates out for readability):

https://gist.github.com/p1d1d1/af240b4261f8d0313d75ea56aae7d96e
This is valid JSON but invalid GeoJSON

You'll find a valid (I guess) GeoJSON here:
https://gist.github.com/p1d1d1/a33b090eefd7f2c883479d59b9e742aa

**Comment by p1d1d1** [comment]
PS: don't know if 2) can be easily fixed

**Comment by procrastinatio** [comment]
1) Removing things will break stuff on clients
2) Is easy (won't break things)
According to RFC7946, GeoJSON is  WGS84 only

**Comment by p1d1d1** [comment]
>1. Removing things will break stuff on clients

Do we have clients that explicitly use ?geometryFormat=geojson? If yes I assume they have GeoJSON based applications and then I don't understand how they could use the API output which is invalid GeoJSON.
Isn't that clients use instead the API default?   

> 2. Is easy (won't break things)
>    According to RFC7946, GeoJSON is  WGS84 only

Niet! It is possible to have other CRSs.

So at the end we don't provide GeoJSON. We just "geojsonize" the coordinates, right?

**Comment by loicgasser** [comment]
> So at the end we don't provide GeoJSON. We just "geojsonize" the coordinates, right?

yes and we put extra parameters in the GeoJSON, but  the parsers usually don't have any problem reading it. we don't return GeometryCollection, instead we we return a list a GeoJSON features which are parsed individually on the client side. But a lot of people are relying on these properties. They should have been in the properties field, but now it's a little late. I would do it for the next version of the API.

**Comment by p1d1d1** [comment]
@loicgasser actually if you "just delete" `{"feature": `and the corresponding closing `}`, the result is considered as valid by geojsonlint

**Comment by loicgasser** [comment]
Ok so extra parameters are tolerated which is a good thing.
